code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -500,11 +500,12 @@
 		C = owner
 		no_update = FALSE
 
-	if(HAS_TRAIT(C, TRAIT_HUSK) && IS_ORGANIC_LIMB(src))
-		dmg_overlay_type = "" //no damage overlay shown when husked
-		is_husked = TRUE
-	else
-		is_husked = FALSE
+	if(IS_ORGANIC_LIMB(src))
```

Comment:
Runtime due to undefined 'status_traits' when loading dummies through select equipment.

Fixed by nesting the checks by 1.